state-reconstruct-fetcher/src/types/v2.rs

```diff
@@ -37,7 +36,7 @@ impl CommitBlockFormat for V2 {
 impl TryFrom<&abi::Token> for V2 {
     type Error = ParseError;
 
-    /// Try to parse Ethereum ABI token into [`CommitBlockInfo`].
+    /// Try to parse Ethereum ABI token.
     ///
     /// * `token` - ABI token of `CommitBlockInfo` type on Ethereum.
```

Comment:
```suggestion
    /// Try to parse Ethereum ABI token into [`V2`].
```

Reply:
why the brackets?

Reply:
It's just a convenient way to link the documentation. https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html